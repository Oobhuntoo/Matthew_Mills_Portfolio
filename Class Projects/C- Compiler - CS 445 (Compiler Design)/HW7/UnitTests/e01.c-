main()
{
    bool x;

    if true then output(13);
    output(23);

    x = false;
    if x then output(44);
    if not x then output(55);

    if 3>1 then output(987);
    if 3<1 then output(211);

    if true then output(13); else output(51);
    if false then output(23); else output(451);

    x = false;
    if x then output(44); else output(514);
    if not x then output(55); else output(1514);

    if 3>1 then output(987); else output(114);
    if 3<1 then output(211); else output(1614);
}
