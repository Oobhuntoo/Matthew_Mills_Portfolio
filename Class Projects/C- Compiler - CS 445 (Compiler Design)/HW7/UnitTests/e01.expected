Loading file: e01.tm
13 23 55 987 13 451 514 55 987 1614 
Status: Halted
Bye.
